apps/meteor/client/views/room/contextualBar/ExportMessages/FileExport.tsx

```diff
@@ -31,10 +31,10 @@ const FileExport = ({ formId, rid, exportOptions, onCancel }: FileExportProps) =
 		[t],
 	);
 
-	const handleExport = ({ type, dateFrom, dateTo, format }: MailExportFormValues) => {
+	const handleExport = ({ dateFrom, dateTo, format }: MailExportFormValues) => {
```

Comment:
If you are changing the props expected, you will need to change where this function is called to... But I think you can do that change in another PR, this is not important from what you are doing here, right?

Reply:
Actually, this is called directly by handleSubmit :thinking: Is there any way to make it not send the type param to this function? I thought of removing type from `MailExportFormValues`, but this is used to set default values.

Reply:
ok, in this case... zero problems